src/components/NcTimezonePicker/NcTimezonePicker.vue

```diff
@@ -81,6 +82,13 @@ export default {
 			type: String,
 			default: 'floating',
 		},
+		/**
+		 * ID of the inner vue-select element, can be used for labels like: `vs-${uid}__combobox`
+		 */
+		uid: {
+			type: [String, Number],
+			default: `tz-${(Math.random() + 1).toString(36).substring(7)}`,
+		},
```

Comment:
`default` must be a function, otherwise all instances have random but the same ID.

BTW, there is `GenRandomId`: https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/src/utils/GenRandomId.js